Zscaler/zscaler-zia/ingest/parser.yml

```
@@ -0,0 +1,122 @@
+          zscaler.zia.category: "{{json_event.message.event.category}}"
+          zscaler.zia.sub_category: "{{json_event.message.event.subcategory}}"
+          zscaler.zia.department: "{{json_event.message.event.department}}"
+          zscaler.zia.audit.log_type: "{{json_event.message.event.auditlogtype}}"
+
+          zscaler.zia.threat.name: "{{json_event.message.event.threatname}}"
+          zscaler.zia.threat.class: "{{json_event.message.event.threatclass}}"
+          zscaler.zia.threat.category: "{{json_event.message.event.threatcategory or json_event.message.event.threatcat}}"
+          zscaler.zia.device.owner: "{{json_event.message.event.deviceowner}}"
+          zscaler.zia.keyprotectiontype: "{{json_event.message.event.keyprotectiontype}}"
+          zscaler.zia.tuntype: "{{json_event.message.event.tuntype}}"
+          zscaler.zia.avgduration: "{{json_event.message.event.avgduration}}"
+
+          http.request.method: "{{json_event.message.event.requestmethod}}"
+          http.request.bytes: "{{json_event.message.event.requestsize}}"
+          http.request.referrer: "{{json_event.message.event.referrerURL}}"
+          http.response.status_code: "{{json_event.message.properties.status}}"
+          http.response.bytes: "{{json_event.message.event.responsesize}}"
+          http.response.mime_type: "{{json_event.message.event.contenttype}}"
+
+          server.ip: "{{json_event.message.event.serverip}}"
+
+          network.application: "{{json_event.message.event.applicationname}}"
+
+          user_agent.original: "{{json_event.message.event.useragent}}"
+
+          file.type: "{{json_event.message.event.filetype}}"
+          file.name: "{{json_event.message.event.object_name_1}}"
+
+          url.domain: "{{json_event.message.event.url}}"
+
+          user.email: "{{json_event.message.event.user or json_event.message.event.login}}"
+          network.protocol: "{{json_event.message.event.protocol or json_event.message.event.proto}}"
+
+          source.ip: "{{json_event.message.event.ClientIP or json_event.message.event.clientip or json_event.message.event.clt_sip or json_event.message.event.csip}}"
+          source.port: "{{json_event.message.event.csport}}"
+          source.bytes: "{{json_event.message.event.inbytes}}"
+
+          destination.port: "{{json_event.message.event.srv_dport or json_event.message.event.sdport}}"
+          destination.ip: "{{json_event.message.event.srv_dip or json_event.message.event.sdip}}"
+          destination.bytes: "{{json_event.message.event.outbytes}}"
+          destination.geo.country_name: "{{json_event.message.event.destcountry}}"
+
+          dns.question.name: "{{json_event.message.event.dns_req}}"
+          dns.response_code: "{{json_event.message.event.dns_resp}}"
+          dns.question.type: "{{json_event.message.event.dns_reqtype}}"
+
+          host.name: "{{json_event.message.event.hostname or json_event.message.event.devicehostname}}"
```

I also suggest setting `event.dataset` from the source type

```suggestion
          host.name: "{{json_event.message.event.hostname or json_event.message.event.devicehostname}}"
              
      - translate:
        dictionary:
          "zscalernss-audit": "audit"
          "zscalernss-casb": "casb"
          "zscalernss-dns": "dns"
          "zscalernss-fw": "firewall"
          "zscalernss-web": "web"
        mapping:
          json_event.message.sourcetype: event.dataset
```